H2MLauncher.UI/MainWindow.xaml

```diff
@@ -81,6 +81,15 @@
                     <ProgressBar Name="pbStatus"  Width="400" Height="30" Value="{Binding UpdateDownloadProgress, Mode=OneWay}"/>
                     <TextBlock Text="{Binding ElementName=pbStatus, Path=Value, StringFormat={}{0:0}%}" HorizontalAlignment="Center" VerticalAlignment="Center" />
                 </Grid>
+                <Grid Grid.Column="1" Width="400" Height="30" Visibility="{Binding ReleaseNotesVisibility}">
```

Comment:
Now i see why you removed the flag. There should be a BooleanToVisibilityConverter (https://learn.microsoft.com/en-us/dotnet/api/system.windows.controls.booleantovisibilityconverter?view=windowsdesktop-8.0), maybe use that and put it in the global resources, we might need that often